src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1076,6 +1076,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('cap_mbps', arg_group='Additional Flags', help="Caps the transfer rate, in megabits per second. "
                    "Moment-by-moment throughput might vary slightly from the cap. "
                    "If this option is set to zero, or it is omitted, the throughput isn't capped. ")
+        c.argument('extra_options', arg_group='Additional Flags',
+                   help="Other options which will be passed through to azcopy as it is")
```

Comment:
Let's include some examples in the help message.

Reply:
If we can't decide which way from #18869 to go, consider mark it as experimental.